packages/cors/README.md

```diff
@@ -92,6 +92,19 @@ app.use(ampCors({
 }));
 ```
 
+
+### Email Mode
+
+AMP for Email has [specific AMP CORS requirements](https://developers.google.com/gmail/ampemail/security-requirements). You can enable the AMP for Email CORS mode via the `email option`: 
```

Comment:
s/AMP for Email/Gmail here, since it's not standardized and will likely be replaced by

Reply:
rephrased